src/main/java/com/twilio/oai/DirectoryStructureService.java

```diff
@@ -72,11 +72,17 @@ public static class ContextResource {
 
     public void configure(final OpenAPI openAPI) {
         final Map<String, DependentResource> versionResources = getVersionResourcesMap();
+        Map<String, PathItem> pathsToSkipMap = new HashMap<>();
 
         isVersionLess = additionalProperties.getOrDefault(API_VERSION, "").equals("");
 
         openAPI.getPaths().forEach(resourceTree::addResource);
         openAPI.getPaths().forEach((name, path) -> {
+            final Optional<String> skipPath = Optional.of(
+                // if the flag is set, this path need not be generated by helper libraries
+                PathUtils.getTwilioExtension(path, "x-skip-operation").orElse(""));
```

Comment:
if` x-skip-operation` is marked as false. Will it skip or keep the path ?

Reply:
It'll keep the path, work as usual.